dev/serializer_context.h

```diff
@@ -8,7 +8,7 @@ namespace sqlite_orm {
             bool replace_bindable_with_question = false;
             bool skip_table_name = true;
             bool use_parentheses = true;
-            bool skip_types_and_constraints = false;
+            bool skip_types_and_constraints_except_unindexed = false;
```

Comment:
Can we rename this flag into something like `fts5_columns`? Also see below when checking it.

Reply:
done